hirola/front/forms/user_forms.py

```diff
@@ -422,3 +423,39 @@ class PhoneProfileUserDataCollectionForm(forms.Form):
 
     CHOICES = (('0', 'Quantity',),)
     quantity = forms.CharField(error_messages=quantity_error_messages)
+
+
+class ContactUsForm(forms.Form):
+
+    error_messages = {
+        'invalid': _('Please enter a valid comment'),
+        'required': _("Comment cannot be empty!")
+    }
+
+    name = forms.CharField(required=False, widget=forms.TextInput(
+        attrs={'placeholder': 'Name(Optional)'}))
+    email = forms.EmailField(widget=forms.TextInput(
+        attrs={'placeholder': 'Email'}))
+    comment = forms.CharField(widget=forms.Textarea(
+        attrs={'placeholder': 'Tell us about the issue..',
+               'class': 'materialize-textarea'}))
+
+    def clean_comment(self):
+        comment = self.cleaned_data.get('comment')
+        if not comment or comment.isspace():
+            raise ValidationError(self.error_messages['required'])
+        if re.match(r'^[_\W]+$', comment):
+            raise ValidationError(self.error_messages['invalid'])
+        return comment
+
+    def send_email(self):
+        to_email = 'testermail717@gmail.com'
+        from_email = self.cleaned_data.get('email')
```

Comment:
The email of the user cannot be seen on the mail. You may have to send it in the body of the message, since this data is getting lost.